src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -231,6 +231,9 @@ The `EvmNode` object includes the following items for a given coin or token:
 
 
 ### ConfirmationTargets
+<DevComment>
+  This section needs descriptions for each of the parameters.
+</DevComment>
```

Comment:
`ConfirmationTargets`: represents the number of blocks required for an on-chain lightning-related transaction to be confirmed. It is used for estimating the transaction fee rate (feerate) for different transaction types in the context of permissionless transactions performed by the node. Different target types are `background`, `normal`, `high_priority`.

`background`: This target is used for transactions that can tolerate slower confirmation times when the transaction fee rate decreases. These transactions are not time-sensitive and can afford to wait longer for confirmation. A recommended range for background could be 12 to 144 blocks (approximately 2 hours to 1 day) to ensure a low feerate.

`normal`: This target is applied to transactions that we want to confirm promptly, without significant delay. For instance, it could be a transaction for opening a payment channel. These transactions are important but not critical. A suggested value for normal could be around 6 blocks (approximately 1 hour) to ensure a moderate feerate.

`high_priority`: This target is used for transactions that require quick confirmation to prevent potential loss of funds. Examples include redeeming a Hashed Time Lock Contract (HTLC) on the blockchain before it times out or penalizing a malicious counterparty. These transactions are time-critical and must be confirmed promptly to ensure the security of funds. A recommended value for high_priority could be around 1-2 blocks (approximately 10-20 minutes) to ensure a high feerate.